models/wrf_hydro/skip_worktree.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I don't the `skip_worktree.sh` script should be committed.
It seems specific to a local git workflow.